src/components/explore-the-community.tsx

```diff
@@ -0,0 +1,19 @@
+import { Button } from "./ui/button";
```

Comment:
Please use absolute imports.

```suggestion
import { Button } from "@/components/ui/button";
```